### PR TITLE
Muzima 794

### PR DESCRIPTION
**Pull request by sthaiya**

### CODE REVIEW

File: app/src/main/java/com/muzima/adapters/relationships/AutoCompleteRelatedPersonAdapter.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    @Override
+    protected List<Person> getOptions(CharSequence constraint) {
+        PatientController patientController = getMuzimaApplicationContext().getPatientController();
+        PersonController personController = getMuzimaApplicationContext().getPersonController();
+        List<Person> personList = new ArrayList<>();
+        try {
+
+            List<Patient> patientList;
+            if (searchRemote) {
+                patientList = patientController.searchPatientOnServer(constraint.toString());
```
Comment (mssavai) [inline app/src/main/java/com/muzima/adapters/relationships/AutoCompleteRelatedPersonAdapter.java]: Looks like you will need to add ability search person on server

File: app/src/main/java/com/muzima/controller/FormController.java [hunk 2]
```diff
@@ -390,6 +390,7 @@ public void saveFormData(FormData formData) throws FormDataSaveException {
         try {
             formData.setSaveTime(new Date());
             formService.saveFormData(formData);
+            System.out.println(formData.getJsonPayload());
```
Comment (mssavai) [inline app/src/main/java/com/muzima/controller/FormController.java]: I think this log is no longer in use.

File: app/src/main/java/com/muzima/controller/FormController.java [hunk 3]
```diff
@@ -713,6 +724,7 @@ public FormDataStatusDownloadException(Throwable throwable) {
     boolean uploadFormDataToServer(List<FormData> allFormData, boolean result) throws IOException {
         for (FormData formData : allFormData) {
             String rawPayload = formData.getJsonPayload();
+            System.out.println(rawPayload);
```
Comment (mssavai) [inline app/src/main/java/com/muzima/controller/FormController.java]: This is also no longer needed.

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -543,6 +550,11 @@
     <string name="info_provider_not_downloaded">Did not download any providers</string>
     <string name="info_provider_unavailable">No providers have been added yet</string>
     <string formatted="false" name="info_providers_downloaded">Downloaded %d providers</string>
+    <string name="info_relationships_delete_success">%d Relationships deleted successfully</string>
+    <string formatted="false" name="info_relationships_download_progress">Downloaded relationships for client %d of %d.</string>
```
Comment (mssavai) [inline app/src/main/res/values/strings.xml]: I noticed there a number of new user messages that need to be translated.

**Comment by sthaiya** [comment]
Please merge similarly as we merged muzimacore

**Pull request merged**